django/researchdata/models.py

```diff
@@ -69,5 +71,25 @@ class Answer(models.Model):
     def __str__(self):
         return str(self.answer_text)[0:40]
 
+    def save(self, *args, **kwargs):
+        """
+        When a new model is saved email the research team
+        """
+
+        # Check if this is a new answer
+        if self.meta_created_datetime is None:
+            # Send email alert to research team
+            try:
+                send_mail('CLiC Calendar: New Answer',
+                          'There has been a new answer submitted to CLiC Calendar.',
+                          settings.DEFAULT_FROM_EMAIL,
+                          [settings.NOTIFICATION_EMAIL],
+                          fail_silently=False)
+            except Exception as e:
+                print(e)
```

Comment:
```suggestion
                logger.exception("Failed to send email")
```

Reply:
The print would go nowhere on a live system, IIRC. But the logger message should go to a log file.